terraform/service/security.tf

```diff
@@ -0,0 +1,33 @@
+# Security Group for the service
+resource "aws_security_group" "allow_rules_service" {
+  name        = "${var.service_subdomain}-allow-rule"
+  description = "Allow inbound traffic for the service"
+  vpc_id      = data.terraform_remote_state.ecs_infrastructure.outputs.vpc_id
+
+  # Allow traffic from ALB to frontend
+  ingress {
+    from_port   = 3000
```

Comment:
These are variables in code we already use, please take terraform from other projects when you can